docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -253,3 +256,23 @@ Creates an exception that excludes all LFC-signed trusted processes:
 
 [role="screenshot"]
 image::images/nested-exp.png[]
+
+[float]
+[[manage-exceptions]]
+=== View and manage exceptions
+
+The Exception lists table enables you to view and manage all exceptions that have been assigned to rules. To view the Exception lists table, go to *Detections* -> *Manage detection rules*, then select the *Exception Lists* tab.
```

Comment:
Should say "view and manage all exception lists".

Reply:
+1 - would be great to get feedback on if users are understanding our lingo with lists. I think we've tried to stay consistent throughout but it does get confusing.

Reply:
@yctercero @peluja1012 -- just adding my thoughts here. IMO, sometimes the language does get a little confusing. In the UI, it says "add a rule exception." `Exception lists` to me makes it sounds like there are multiple lists of exceptions, when all the exceptions are combined into a single list -- at least that's how it appears to me in the UI. `Exceptions list` sounds more accurate to what the UI is displaying. Perhaps I'm not understanding the feature completely but from a user perspective, that's what I see.

Reply:
Yea, I see what you're saying. Prior to adding this exception lists table, I could definitely see how it would appear to the user that there is only one exception list per rule when in reality multiple exception lists can be assigned to a single rule. It's probably even more confusing because there isn't yet complete exception list management available in the UI. 

I assume that the more accurate functionality with exception lists is that you have numerous lists, any of which you can assign to 'x' number of rules. It's probably more a product question of how we want our users to understand the relationship between exception lists and rules.